package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@discord/embedded-app-sdk",
-  "version": "1.7.0",
+  "version": "1.7.1",
```

Comment:
nit - we don't bump the package version in the PR, it happens automatically via the `release please` github action.

nit - I think this is a feature, not a patch

Reply:
Ah, I didn't realize! I have reverted to 1.7.0